# API: trace()
# Bug description: Trace sanity check fails when using autocast on Conv node
#                  when using torch.jit.trace in PyTorch v1.12.1
import torch
def f(x):
  return x*2
z = torch.jit.trace(f, (torch.zeros(10),))
print(type(z))
